keps/sig-apps/0753-sidecarcontainers.md

```diff
@@ -86,7 +139,25 @@ Check these off as they are completed for the Release Team to track. These check
 
 ## Summary
 
-To solve the problem of container lifecycle dependency we can create a new class of container: a "sidecar container" that behaves primarily like a normal container but is handled differently during termination and startup.
+This KEP adds the concept of sidecar containers to Kubernetes. This KEP proposes
+to add a `lifecycle.type` field to the `container` object in the `pod.spec` to
+define if a container is a sidecar container. The only valid value for now is
+`sidecar`, but other values can be added in the future if needed.
+
+Pods with sidecar containers only change the behaviour of the startup and
+shutdown sequence of a pod: sidecar containers are started before non-sidecars
+and stopped after non-sidecars.
```

Comment:
I'd emphasize here that they forcefully stopped when all other containers finished. E.g.:

```suggestion
and stopped after non-sidecars. Sidecar containers are also terminated when no non-sidecar containers finished. 
```

Reply:
Good point, thanks! Will amend to current patches, if that is okay to you :)